common/persistence/client/fault_injection_errorgenerator.go

```diff
@@ -109,12 +109,12 @@ func (p *DefaultErrorGenerator) UpdateRate(rate float64) {
 	p.Lock()
 	defer p.Unlock()
 
-	p.rate.Store(rate)
+	p.rate.Store(math.Float64bits(rate))
```

Comment:
Is it better to convert to int vs having `atomic.Pointer[float64]`?
Also, is the std `atomic` module any better than the `uber` one? Or is it just to reduce deps?

Reply:
Just reduce deps mostly, uber's atomic wraps `sync/atomic`